log/logging.go

```diff
@@ -7,26 +7,24 @@ package log
 import (
 	"fmt"
 	"os"
-
-	"github.com/sirupsen/logrus"
-
-	"github.com/weaveworks/promrus"
 )
 
-// Setup configures a global logrus logger to output to stderr.
-// It populates the standard logrus logger as well as the global logging instance.
+// Setup configures a global Gokit logger to output to stderr.
+// It populates the Gokit logger as well as the global logging instance.
 func Setup(logLevel string) error {
-	level, err := logrus.ParseLevel(logLevel)
+	var level Level
+	err := level.Set(logLevel)
 	if err != nil {
 		return fmt.Errorf("error parsing log level: %v", err)
 	}
-	hook, err := promrus.NewPrometheusHook() // Expose number of log messages as Prometheus metrics.
+
+	// TODO understand whether this is needed
```

Comment:
Can you investigate this, please?

Reply:
This method is not used by any of the projects (`dskit`, `Mimir`, `Loki`, `Tempo`, `Pyroscope`), so I decided to delete it.

Reply:
It is now deleted. I was sure I did it, but somehow I missed it.